src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -1136,9 +1136,9 @@ class OPENASSETIO_CORE_EXPORT Manager {
    * the Manager may even use this opportunity to switch to some
    * temporary working path or some such.
    *
-   * @warning If the supplied @ref trait_set is missing traits required
-   * by the manager for any input entity reference, then that element
-   * will error.
+   * @warning If the supplied @fqref{TraitsData} "trait data" is missing
```

Comment:
[Consider] `@fqref{TraitsDatas} "traits datas"` ?

Reply:
The sentence is referring to the `TraitsData` for a particular reference, so I _think_ the grammar is correct.  Tricky with all these plurals of plurals flying around.

Reply:
Not specifically relevant for this concern, but updated that paragraph to have a link to the glossary in daa8a1ac.